editoast/src/views/v2/train_schedule.rs

```diff
@@ -934,4 +998,49 @@ mod tests {
         let response = call_service(&service, request).await;
         assert!(response.status().is_success());
     }
+
+    #[rstest]
+    #[ignore] // TODO: This test should be rewritten using mocks
+    async fn train_schedule_simulation_summary(
+        #[future] timetable_v2: TestFixture<Timetable>,
+        #[future] small_infra: TestFixture<Infra>,
+        db_pool: Data<DbPool>,
+    ) {
+        let timetable = timetable_v2.await;
+        let infra = small_infra.await;
+        let rolling_stock =
+            named_fast_rolling_stock("simulation_summary_rolling_stock", db_pool.clone()).await;
+
+        let train_schedule_base: TrainScheduleBase = TrainScheduleBase {
+            rolling_stock_name: rolling_stock.name.clone(),
+            ..serde_json::from_str(include_str!("../../tests/train_schedules/simple.json"))
+                .expect("Unable to parse")
+        };
+        let train_schedule = TrainScheduleForm {
+            timetable_id: timetable.id(),
+            train_schedule: train_schedule_base,
+        };
+        let request = TestRequest::post()
+            .uri("/v2/train_schedule")
+            .set_json(json!(vec![train_schedule]))
+            .to_request();
+
+        let service = create_test_service().await;
+        let train_schedule: Vec<TrainScheduleResult> =
+            call_and_read_body_json(&service, request).await;
+        assert_eq!(train_schedule.len(), 1);
+        let request = TestRequest::get()
+            .uri(
+                format!(
+                    "/v2/train_schedule/simulation_summary/?infra={}&ids[]={}",
+                    infra.id(),
+                    train_schedule[0].id,
+                )
+                .as_str(),
+            )
+            .to_request();
+
+        let response = call_service(&service, request).await;
+        assert!(response.status().is_success());
```

Comment:
Can we/Should we assert more than just a **success**?

Reply:
the tests will be rewritten